src/engine/renderva.cpp

```diff
@@ -1338,11 +1339,10 @@ void renderzpass(renderstate &cur, vtxarray *va)
     if(cur.vbuf!=va->vbuf) changevbuf(cur, RENDERPASS_Z, va);
     if(!cur.depthmask) { cur.depthmask = true; glDepthMask(GL_TRUE); }
     if(cur.colormask) { cur.colormask = false; glColorMask(GL_FALSE, GL_FALSE, GL_FALSE, GL_FALSE); }
-    int firsttex = 0, numtris = va->tris;
+    int numtris = va->tris;
```

Comment:
Whenever you change such declarations, please consider using `auto`.

Reply:
Could you be a bit more specific? `va->tris` is an `int` here, could not find any reason to `auto` this unless it was some long/annoying type to write over and over (and `numtris` seems kind of self-explanatory)

Reply:
This is just a general suggestion I had while reading through your code. `auto` just makes sure no narrowing can ever happen in the future. But I guess `-Wall` helps us avoid this problem.